File: servico-musica-java/java.graphql/src/main/java/servico/java/graphql/repository/PlaylistRepository.java
package servico.java.graphql.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import servico.java.graphql.entity.Playlist;

import java.util.List;

@Repository
public interface PlaylistRepository extends JpaRepository<Playlist,Long> {
    List<Playlist> findByMusicasId(Long idMusica);
}

File: servico-musica-kotlin/kotlin.graphql/src/main/kotlin/servico/kotlin/graphql/repository/PlaylistRepository.kt
package servico.kotlin.graphql.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import servico.kotlin.graphql.entity.Playlist

@Repository
interface PlaylistRepository : JpaRepository<Playlist, Long> {
    fun findByMusicasId(idMusica: Long?): MutableList<Playlist?>?
}
